test/collection_test.js

```diff
@@ -1167,6 +1167,33 @@ describe("Collection", () => {
     });
   });
 
+  /** @test {Collection#deleteAll} */
+  describe("#deleteAll", () => {
+    let articles;
+
+    beforeEach(() => {
+      //Create 5 Records
+      articles = testCollection();
+      articles.create(article);
+      articles.create(article);
+      articles.create(article);
+      articles.create(article);
+      articles.create(article);
+      return articles;
+    });
+
+    it("should be able to soft delete all articles", () => {
+      return articles.deleteAll().then(res => {
+        res.data.forEach(record => {
+          articles
+            .get(record.id)
+            .then(res => res.data._status)
+            .should.eventually.eql("deleted");
+        });
+      });
```

Comment:
I believe this test could be simplified by just testing `deleteAll()` → `list()` returns 0 elements → `list({includeDeleted: true})` returns 5 elements